server/bleep/src/webserver/studio.rs

```diff
@@ -451,6 +451,74 @@ pub struct Import {
     pub thread_id: Uuid,
 }
 
+async fn extract_relevant_chunks(
+    app: Application,
+    exchanges: &[Exchange],
+    context: &[ContextFile],
+) -> webserver::Result<Vec<ContextFile>> {
+    let context_json = serde_json::to_string(&context).unwrap();
+
+    let answer_api_token = app
+        .answer_api_token()
+        .map_err(|e| Error::user(e).with_status(StatusCode::UNAUTHORIZED))?
+        .map(|s| s.expose_secret().clone());
+
+    // Create an instance of the LLM gateway client
+    let llm_gateway = llm_gateway::Client::new(&app.config.answer_api_url)
+        .temperature(0.0)
+        .bearer(answer_api_token);
```

Comment:
We can specify the LLM model here to ensure it uses the same model as other studio calls.
```suggestion
    let llm_gateway = llm_gateway::Client::new(&app.config.answer_api_url)
        .model(LLM_GATEWAY_MODEL)
        .temperature(0.0)
        .bearer(answer_api_token);
```